src/ResourceManager/Compute/Commands.Compute/help/New-AzureRmGalleryImageVersion.md

```diff
@@ -0,0 +1,290 @@
+---
+external help file: Microsoft.Azure.Commands.Compute.dll-Help.xml
+Module Name: AzureRM.Compute
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.compute/new-azurermgalleryimageversion
+schema: 2.0.0
+---
+
+# New-AzureRmGalleryImageVersion
+
+## SYNOPSIS
+Create a gallery image version.
+
+## SYNTAX
+
+```
+New-AzureRmGalleryImageVersion [-ResourceGroupName] <String> [-GalleryName] <String>
+ -GallleryImageDefinitionName <String> [-GalleryImageVersionName] <String> [-AsJob] [-Location] <String>
+ [-SourceImageId] <String> [-Tag <Hashtable>] [-ReplicaCount <Int32>] [-PublishingProfileExcludeFromLatest]
+ [-PublishingProfileEndOfLifeDate <DateTime>] [-TargetRegion <Hashtable[]>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Create a gallery image version.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> $region1 = @{Name='West US';ReplicaCount=1}
+PS C:\> $region2 = @{Name='East US';ReplicaCount=2}
+PS C:\> $region3 = @{Name='Central US'}
+PS C:\> $targetRegions = @($region1,$region2,$region3)
+PS C:\> New-AzureRmGalleryImageVersion -ResourceGroupName $rgname -GalleryName $galleryName -GalleryImageName $imageName -Name $versionName -Location $location -SourceImageId $sourceImageId -ReplicaCount 2 -PublishingProfileExcludeFromLatest -PublishingProfileEndOfLifeDate $endOfLifeDate -TargetRegion $targetRegions
+```
+
+Create a gallery image version.
+
+## PARAMETERS
+
+### -AsJob
+Run cmdlet in the background
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -GalleryImageVersionName
+The name of the gallery image version
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases: Name
```

Comment:
@hyonholee same comment here: swap name and alias and keep everything else the same